docker-compose.yaml

```diff
@@ -17,11 +17,14 @@ services:
       - postgresql
     environment:
       # Basic Configuration
+      EDC_CONNECTOR_NAME: 'example-connector'
+      EDC_IDS_ID: urn:connector:example-connector
```

Comment:
can it be generated from EDC_CONNECTOR_NAME?

Reply:
only if we have another layer of env variables, and I'd wait to set those in our dockerfile rather than add a .env to the docker-compose.yaml